src/quo/components/drawers/drawer_action/view.cljs

```diff
@@ -0,0 +1,86 @@
+(ns quo.components.drawers.drawer-action.view
+  (:require
+    [quo.components.drawers.drawer-action.style :as style]
+    [quo.components.icon :as icon]
+    [quo.components.markdown.text :as text]
+    [quo.components.selectors.selectors.view :as selectors]
+    [quo.theme]
+    [react-native.core :as rn]
+    [schema.core :as schema]))
+
+(def ?schema
```

Comment:
don't we move schema in the separate ns ? cc @J-Son89

Reply:
wasn't sure if we put a hard rule on that.
My understanding was just if the file was becoming too large or because the schema itself was too large then we should move it out. 
Sounds good to me anyway, @clauxx maybe you know about this? 🤔

Reply:
Yeah its not necessary to put it outside. If its small-ish and it's just one schema var i guess it's fine.